components/Map.js

```diff
@@ -1,21 +1,72 @@
-import { MapContainer, TileLayer, ZoomControl } from "react-leaflet";
+import {
+  MapContainer,
+  TileLayer,
+  ZoomControl,
+  Marker,
+  Popup,
+} from "react-leaflet";
+import L from "leaflet";
 import "leaflet/dist/leaflet.css";
 import styled from "styled-components";
 
-export default function Map() {
+const markerIcon = L.divIcon({
```

Comment:
I would code the const markerIcon in a markerIcon.js and import it hier.
It is a huge svg that nobody can understand and cares about  😅